content/_data/changelogs/weekly.yml

```diff
@@ -12911,6 +12911,187 @@
       - type: rfe
         message: Always allow configuring agent-to-controller security subsystem.
 
+  - version: '2.320'
+    date: 2021-11-09
+    changes:
+      - type: rfe
+        category: developer
+        pull: 5856
+        authors:
+          - basil
+          - timja
+        pr_title: Unfork `AntClassLoader`
+        message: |-
+          Developer: Jenkins now uses the upstream version of <code>AntClassLoader</code> without custom patches.
+      - &1
```

Comment:
not sure what's going on here, I assume it'll work but not sure why

Reply:
We didn't understand this one either, but are confident that we can correct it after the release.